### ISSUE TITLE
Feature Request: Updating typings.

### ISSUE THREAD

**Issue by rkralston** [angular/angular-cli#1265]
Upgrade typings to 1.3.0

New tsconfig:

{
  "compileOnSave": false,
  "compilerOptions": {
    "declaration": false,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "module": "commonjs",
    "moduleResolution": "node",
    "noEmitOnError": true,
    "noImplicitAny": false,
    "outDir": "../dist/",
    "rootDir": ".",
    "sourceMap": true,
    "target": "es5",
    "inlineSources": true
  },

  "files": [
    "../typings/index.d.ts"
  ]
}

New typings.d.ts:

declare var module: { id: string };

Rerun npm i.

ambient is gone. Everything is moved to globals. index.d.ts has references to all d.ts files.

ng build works.

Thanks to my Ops group for needling this out.

**Comment by filoxo** [comment]
I achieved the same thing without modifying `tsconfig.json`.

- Edit `typings.json` and replace all references to *ambient* to *global* (eg. *ambientDependencies* becomes *globalDependencies*).
- Update the ref path in `typings.d.ts` to use `index.d.ts` of `browser.d.ts`. 

        /// <reference path="../typings/index.d.ts" />

- Delete `typings/` directory
- Run `typings install`
- Grab your drink of choice, and begin resolving issues with typings. For me, I had to move 3 dependencies into globalDependencies and I have no idea why. ¯\\\_(ツ)_/¯
